Makefile

```diff
@@ -29,9 +29,8 @@ RPM_BUILD_OPTIONS := --with=mysql
 
 OSUSE_REPOS = https://download.opensuse.org/repositories
 ifeq ($(REPOSITORY_URL),)
-ifneq ($(ID),centos)
-ADD_REPOS = "munge"
-endif
+SLES_12_PR_REPOS = munge
```

Comment:
Why is this coming from Jenkins and not from Nexus?  As a general rule, *PR_REPOS should only be used if you want to pull an RPM from a PR, to test (with) it for example.

Reply:
If a repository server is not configured, then MUNGE on sles-12.x/leap-42.x must be pulled from the latest jenkins job for munge.

What variable to I now set to cause that to happen?

Reply:
> If a repository server is not configured, then MUNGE on sles-12.x/leap-42.x must be pulled from the latest jenkins job for munge.

I don't think we can (or should try to) support an environment without a repo server.  It has caused us enough grief not having one that I don't think we should go to any efforts to try to support it for any other unknown environments.  Such support could be a welcome contribution if somebody wanted to work out how to do it and have the flexibility for ABI breaking packages, etc.

Reply:
Well we certainly won't be testing running with out one any time soon.
However this was the pre-repository sever way of getting the munge RPMs, so I was leaving that functionality intact.

Reply:
I don't think we will ever be testing not using a repo server.  It's just not a workable solution.

That said, there is no reason to have the `PR_REPO` used here and it only adds confusion about why it is there when everything else is coming from Nexus.

Reply:
Removed it.